angular.json

```diff
@@ -41,8 +41,9 @@
       }
     }
   },
-  "defaultProject": "applicationinsights-angularplugin-js",
   "cli": {
-    "defaultCollection": "@angular-eslint/schematics"
+    "schematicCollections": [
```

Comment:
This was renamed in v14 

https://github.com/angular/angular-cli/blob/main/CHANGELOG.md#angularcli-8